packages/addons/addon-depends/icu/package.mk

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="icu"
+PKG_VERSION="67-1"
+PKG_SHA256="571d3723a8055d85e513b19b70b4f7f370da10df084c3bb598e0fb8333da0a16"
+PKG_LICENSE="https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE"
+PKG_SITE="http://www.icu-project.org"
+PKG_URL="https://github.com/unicode-org/icu/archive/release-${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_HOST="toolchain:host"
+PKG_DEPENDS_TARGET="toolchain icu:host"
+PKG_LONGDESC="International Components for Unicode library."
+
+PKG_BUILD_FLAGS="-sysroot"
+
+configure_package() {
+  PKG_CONFIGURE_SCRIPT="${PKG_BUILD}/icu4c/source/configure"
+  PKG_CONFIGURE_OPTS_TARGET="--disable-extras \
+                             --disable-renaming \
```

Comment:
```suggestion
                             --disable-layoutex \
                             --disable-renaming \
```

Reply:
I will disable layout and layoutex according to [this](https://github.com/unicode-org/icu/blob/2e56c4264acac327b4a9064a6000c39b602ad833/docs/userguide/layoutengine/paragraph.md)

Reply:
Done